src/main/java/org/snomed/aag/rest/AcceptanceController.java

```diff
@@ -44,26 +48,35 @@ public AcceptanceController(BranchSecurityService securityService,
             @ApiResponse(code = 403, message = "&bull; When the Branch cannot be found from the given branch path. <br/> &bull; When individual Criteria Items do not exist for the branch's expected acceptance criteria."),
             @ApiResponse(code = 200, message = "When the branch (or its parent) has acceptance criteria.", response = ProjectAcceptanceCriteriaDTO.class)
     })
-    @GetMapping("/{branch}")
-    public ResponseEntity<?> viewCriteriaItems(@ApiParam("The branch path.") @PathVariable(name = "branch") String branch) throws RestClientException {
-        branch = BranchPathUriUtil.decodePath(branch);
+	@GetMapping("/{branchPath}")
+	public ResponseEntity<?> viewCriteriaItems(@ApiParam("The branch path.") @PathVariable(name = "branchPath") String branchPath,
```

Comment:
If the request parameter `matchAuthorFlags` is true, only Criteria Items that have a corresponding flag in the Branch's metadata will be returned.